src/main/java/com/practice/test/IncomeTaxCalculatorServiceImp.java

```diff
@@ -21,23 +21,29 @@ public CalculatorTaxResponse incomeTaxCalculation(CalculatorTaxRequest request)
 
             incomeTax = incomeTaxCalculation(request.getIncomeAmount());
         }
-        if (request.getBusinessType() == BusinessTypes.MostCompanies.value || request.getBusinessType() == BusinessTypes.NonProfit.value){
+        else if (request.getBusinessType() == BusinessTypes.MostCompanies.value || request.getBusinessType() == BusinessTypes.NonProfit.value){
             incomeTax = request.getIncomeAmount()*.28f;
 
         }
-        if (request.getBusinessType() == BusinessTypes.MaoriAuthorities.value){
+        else if (request.getBusinessType() == BusinessTypes.MaoriAuthorities.value){
             incomeTax = request.getIncomeAmount()*.175f;
 
         }
 
-        if (request.getBusinessType() == BusinessTypes.TrustsAndTrusteesEarned.value){
+        else if (request.getBusinessType() == BusinessTypes.TrustsAndTrusteesEarned.value){
             incomeTax = request.getIncomeAmount()*.33f;
 
         }
-        if (request.getBusinessType() == BusinessTypes.TrustsAndTrusteesInitial.value){
+        else if (request.getBusinessType() == BusinessTypes.TrustsAndTrusteesInitial.value){
             incomeTax = request.getIncomeAmount()*0.00f;
 
         }
+        else {
```

Comment:
1. Instead of comparing string we can parse the business type value  and compare enum by enum 
2.  we need to validate errors on top of the function rather than at the button this is best practice